### PR TITLE
Changed the staging bridging

### PR DESCRIPTION
**Pull request by KamranMuniree**

### CODE REVIEW

File: helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_staging_eu.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ bridges:
 # todo remove all bridges once we are 100% live and methode publishes directly to this staging. 
 ##CMS
 - name: cms-kafka-bridge-pub-staging-eu
```
Comment (tosan88) [inline helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_staging_eu.yaml]: Could you please rename these modified bridges to reflect that they connect now to prod?

**Comment by sbuliarca** [comment]
I just brought the discussion in upp-tech, let's see what is the outcome from there

**Comment by sbuliarca** [comment]
So, after the talks in upp-tech, we'll need to bridge the following way:
- staging-publish-eu -> prod-publish-eu
- staging-publish-us -> prod-publish-us
- staging-delivery-eu -> staging-publish-eu, staging-publish-us
- staging-delivery-us -> staging-publish-eu, staging-publish-us

**Comment by KamranMuniree** [comment]
@sbuliarca I have now made the amendments

File: helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_staging_eu.yaml (hunk 1 above)
Comment (KamranMuniree) [reply helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_staging_eu.yaml]: I have now made these changes.

**Pull request merged**